lib/ae_mdw/db/name.ex

```diff
@@ -71,6 +72,24 @@ defmodule AeMdw.Db.Name do
     end
   end
 
+  def account_pointer_at(plain_name, time_reference_txi) do
+    case locate(plain_name) do
+      {nil, _module} ->
+        Log.warn("missing name for plain name #{plain_name}")
+        {:error, :name_not_found}
+
+      {m_name, _module} ->
+        case pointee_at(m_name, time_reference_txi) do
```

Comment:
Perhaps we can make `pontee_at` return `{:ok, pk} | {:error, :pointee_not_found}` and leave the call only in this clause

Reply:
good idea, eliminates one case.